Trimble.Modus.Components/Controls/TMRadioButton/TMRadioButton.cs

```diff
@@ -0,0 +1,244 @@
+﻿using Trimble.Modus.Components.Contant;
+using Trimble.Modus.Components.Enums;
+
+namespace Trimble.Modus.Components
+{
+    public class TMRadioButton : ContentView, IDisposable
+    {
+        #region Fields
+
+        private readonly Label _label;
+        private readonly Image _radio;
```

Comment:
Can you rename this member name? like '_icon'?

Reply:
Done